pyproject.toml

```diff
@@ -60,6 +60,9 @@ Jinja2 = { version = "3.1.4", optional = true }
 urllib3 = { version = "2.2.2", optional = true }
 lxml_html_clean = { version = "*", optional = true }
 
+[tool.poetry.overrides]
+utilix = "*"
```

Comment:
I do not think you will need this.

Reply:
Right this is redundant as the overriding actually happens in `pytest.yml`